EIPS/eip-2315.md

```diff
@@ -149,11 +147,23 @@ Bytecode: `0x600556b2b75b6003b3`
 |    5  |   JUMPDEST  |    1 |        [] |        [] |
 |    6  |      PUSH1  |    3 |        [] |        [] |
 |    8  |    JUMPSUB  |    8 |       [3] |        [] |
-|    3  |   BEGINSUB  |    1 |        [] |      [ 8] |
 |    4  |  RETURNSUB  |    2 |        [] |      [ 8] |
 |    9  |       STOP  |    0 |        [] |        [] |
 
-Consumed gas: `26`
+Consumed gas: `25`
+
+### Error on walk-into-subroutine
+
+In this example, the code 'walks' into a subroutine, which is not allowed, and causes an error
+
+|  Pc   |      Op     | Cost |   Stack   |   RStack  |
+|-------|-------------|------|-----------|-----------|
+|    0  |   BEGINSUB  |    1 |        [] |        [] |
+
+
+```
+Error: at pc=0, op=BEGINSUB: invalid subroutine entry
```

Comment:
Do we want an OOG message or another like `invalid subroutine entry` ? 
If we want to use this new message maybe we need to update this part https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2315.md#beginsub ?